docs/reference/release-notes/6.7.0.asciidoc

```diff
@@ -0,0 +1,515 @@
+* SQL: fix object extraction from sources {pull}37502[#37502] (issue: {issue}37364[#37364])
+* SQL: Fix issue with field names containing "." {pull}37364[#37364] (issue: {issue}37128[#37128])
+* SQL: Fix bug regarding alias fields with dots {pull}37279[#37279] (issue: {issue}37224[#37224])
+* SQL: Proper handling of COUNT(field_name) and COUNT(DISTINCT field_name) {pull}37254[#37254] (issue: {issue}30285[#30285])
+* SQL: fix COUNT DISTINCT filtering {pull}37176[#37176] (issue: {issue}37086[#37086])
+* SQL: Fix issue with wrong NULL optimization {pull}37124[#37124] (issue: {issue}35872[#35872])
+* SQL: Fix issue with complex expression as args of PERCENTILE/_RANK {pull}37102[#37102] (issue: {issue}37099[#37099])
+* SQL: Handle the bwc Joda ZonedDateTime scripting class in Painless {pull}37024[#37024] (issue: {issue}37023[#37023])
+* SQL: Fix bug regarding histograms usage in scripting {pull}36866[#36866]
+* SQL: Fix issue with always false filter involving functions {pull}36830[#36830] (issue: {issue}35980[#35980])
+* SQL: protocol returns ISO 8601 String formatted dates instead of Long for JDBC/ODBC requests {pull}36800[#36800] (issue: {issue}36756[#36756])
+* SQL: Enhance Verifier to prevent aggregate or grouping functions from {pull}36799[#36799] (issue: {issue}36798[#36798])
+
+Search::
+* Fix fetch source option in expand search phase {pull}37908[#37908] (issue: {issue}23829[#23829])
+* Throw if two inner_hits have the same name {pull}37645[#37645] (issue: {issue}37584[#37584])
+* Ensure either success or failure path for SearchOperationListener is called {pull}37467[#37467] (issue: {issue}37185[#37185])
+* Use executor `SAME` to handle search related handlers {pull}37427[#37427] (issues: {issue}33732[#33732], {issue}37392[#37392])
+
+Security::
+* Fix exit code for Security CLI tools  {pull}37956[#37956] (issue: {issue}37841[#37841])
+* Fix potential NPE in UsersTool {pull}37660[#37660]
+
+Snapshot/Restore::
+* Streamline S3 Repository- and Client-Settings {pull}37393[#37393]
+* SNAPSHOTS: Upgrade GCS Dependencies to 1.55.0 {pull}36634[#36634] (issues: {issue}35229[#35229], {issue}35459[#35459])
+
+Suggesters::
+* Fix duplicate removal when merging completion suggestions {pull}36996[#36996] (issue: {issue}35836[#35836])
+
+Task Management::
+* Un-assign persistent tasks as nodes exit the cluster {pull}37656[#37656]
+
+ZenDiscovery::
+* Always return metadata version if metadata is requested {pull}37674[#37674]
+
+
+
+[[regression-6.7.0]]
+[float]
+=== Regressions
+
+Infra/Core::
+* Speed up converting of temporal accessor to zoned date time {pull}37915[#37915] (issue: {issue}37826[#37826])
+
+
+
+[[upgrade-6.7.0]]
+[float]
+=== Upgrades
+
+Infra/Core::
+* Use DateFormatter in monitoring instead of joda code {pull}38309[#38309]
+* Use dateformatter in ingest-common to log deprecations {pull}38099[#38099]
```

Comment:
I'm not sure these should be labelled with `upgrade` since they are not changing the version of a dependency. `>enhancement` or `>bug` seems more appropriate for these?

Reply:
Makes sense. I've relabelled the PRs accordingly.